docs/developer-docs/latest/development/admin-customization.md

```diff
@@ -503,14 +505,19 @@ To disable notifications about new Strapi releases, set the `config.notification
 
 #### Theme extension
 
-To extend the theme, use the `config.theme` key.
+Strapi applications can be displayed either in Light or Dark mode (see [administrator profile setup in the User Guide](/user-docs/latest/getting-started/introduction.md#setting-up-your-administrator-profile)), and both can be extended through custom theme development.
+
+To extend the theme, use either:
+
+- the `config.theme.light` key for the Light mode
+- the `config.theme.dark` key for the Dark mode
```

Comment:
I'd omit that and add a comment in the above code example instead.

Reply:
I don't think a comment in the code example should replace this explanation, especially since otherwise there's no proper information about how to extend the theme in the "Theme extension" section. I'll leave this here. But I integrated all you other suggestions, thank you for your review!